processes/troubleshooting/all-purpose-mess-remover.dot

```diff
@@ -7,10 +7,11 @@ digraph {
   # Always start with a restart:
   restart_cromwell_instance [shape=oval label="Restart Cromwell's 'writer' instance"];
 
-  determine_time [shape=oval label="Determine what time things started going wrong"];
-  determine_submissions_of_interest [shape=oval label="Determine a submission of interest from around that time"];
+  download_submission_print_out [shape=oval label="Download submission_print_out.sh from github repo dsp-scripts/cromwell"];
+  make_sure_gcloud_use_firecloud [shape=oval label="Make sure gcloud is using a firecloud.org account configuration"];
+  determine_submissions_of_interest [shape=oval label="Determine a submission of interest at a time right before alerts are generated.\n\nRun the script as\n\n./submission_print_out.sh <env>\n\nSubstitute <env> with environment { dev, prod }.\n\nThe script outputs submission details in the format\n\n<submission timestamp> <submitter email> <namespace> <submission id>"];
```

Comment:
TOL Not sure if this instruction should be so much more detailed than the others?

Reply:
Think we should remove output format section and add it as header to script output